<?xml version="1.0" ?>
<launch>
    <node name="joystick_publisher" pkg="franka_joystick_control" type="joystick_publisher" output="screen" />
    <node name="franka_joystick_control" pkg="franka_joystick_control" type="franka_joystick_control" output="screen" />

</launch>
